pkg/api/admin/openshiftcluster_validatestatic_test.go

```diff
@@ -468,20 +468,6 @@ func TestOpenShiftClusterStaticValidateDelta(t *testing.T) {
 			modify:  func(oc *OpenShiftCluster) { oc.Properties.StorageSuffix = "invalid" },
 			wantErr: "400: PropertyChangeNotAllowed: properties.storageSuffix: Changing property 'properties.storageSuffix' is not allowed.",
 		},
-		{
```

Comment:
@petrkotas why one bing removeD?

Reply:
Thanks for this one. It is linked to my old workflow, which I replaced. But this PR still had it.